src/state/store.ts

```diff
@@ -8,42 +8,42 @@ import { CartItem, Item } from '../types';
 interface State {
   readonly provider: webln.NostrWebLNProvider | undefined;
   readonly cart: CartItem[];
-  ndk: NDK;
+  ndk: NDK | undefined;
 }
 
 interface Actions {
   setProvider(provider: webln.NostrWebLNProvider | undefined): void;
   addItemToCart(item: Item): void;
   removeItemFromCart(item: Item): void;
   clearCart(): void;
-  addRelay(relay: string): void;
```

Comment:
these 2 methods were supposed to address this TODO: https://github.com/getAlby/pos/blob/fb3511a5eacfd401de8c60b4c14611a357c9d46e/src/constants.ts#L5

Reply:
@sepehr-safari I removed them since they are not actually used (and I think we need to consider the UX there)